diff --git a/core/lib/Drupal/Core/Render/Element.php b/core/lib/Drupal/Core/Render/Element.php
index c80ed26..0a54201 100644
--- a/core/lib/Drupal/Core/Render/Element.php
+++ b/core/lib/Drupal/Core/Render/Element.php
@@ -50,7 +50,7 @@ public static function properties(array $element) {
    *   TRUE if the element is a child, FALSE otherwise.
    */
   public static function child($key) {
-    return (!isset($key === '') || (0 != strpos($key, '#')));
+    return (!isset($key === '' || (0 != strpos($key, '#')));
   }
 
   /**
