diff --git a/core/lib/Drupal/Core/Security/DoTrustedCallbackTrait.php b/core/lib/Drupal/Core/Security/DoTrustedCallbackTrait.php
index 5b7f56a..8e3577b 100644
--- a/core/lib/Drupal/Core/Security/DoTrustedCallbackTrait.php
+++ b/core/lib/Drupal/Core/Security/DoTrustedCallbackTrait.php
@@ -96,7 +96,6 @@ public function doTrustedCallback(callable $callback, array $args, $message, $er
         throw new UntrustedCallbackException($message);
       }
     }
-    // @TODO Allow named arguments in https://www.drupal.org/node/3174150
     return call_user_func_array($callback, array_values($args));
   }
 
