diff --git a/core/themes/stable/templates/admin/language-negotiation-configure-form.html.twig b/core/themes/stable/templates/admin/language-negotiation-configure-form.html.twig
index ae46f50..3fcf236 100644
--- a/core/themes/stable/templates/admin/language-negotiation-configure-form.html.twig
+++ b/core/themes/stable/templates/admin/language-negotiation-configure-form.html.twig
@@ -4,6 +4,7 @@
  * Theme override for a language negotiation configuration form.
  *
  * Available variables:
+ * - children: A form items for all groups.
  * - language_types: A list of language negotiation types. Each language type
  *   contains the following:
  *   - type: The machine name for the negotiation type.
@@ -12,9 +13,7 @@
  *     operates.
  *   - configurable: A radio element to toggle the table.
  *   - table: A draggable table for the language detection methods of this type.
- *   - children: Remaining form items for the group.
  *   - attributes: A list of HTML attributes for the wrapper element.
- * - children: Remaining form items for all groups.
  *
  * @see template_preprocess_language_negotiation_configure_form()
  */
diff --git a/core/themes/stable9/templates/admin/language-negotiation-configure-form.html.twig b/core/themes/stable9/templates/admin/language-negotiation-configure-form.html.twig
index ae46f50..3fcf236 100644
--- a/core/themes/stable9/templates/admin/language-negotiation-configure-form.html.twig
+++ b/core/themes/stable9/templates/admin/language-negotiation-configure-form.html.twig
@@ -4,6 +4,7 @@
  * Theme override for a language negotiation configuration form.
  *
  * Available variables:
+ * - children: A form items for all groups.
  * - language_types: A list of language negotiation types. Each language type
  *   contains the following:
  *   - type: The machine name for the negotiation type.
@@ -12,9 +13,7 @@
  *     operates.
  *   - configurable: A radio element to toggle the table.
  *   - table: A draggable table for the language detection methods of this type.
- *   - children: Remaining form items for the group.
  *   - attributes: A list of HTML attributes for the wrapper element.
- * - children: Remaining form items for all groups.
  *
  * @see template_preprocess_language_negotiation_configure_form()
  */
