diff --git a/core/modules/views/src/Form/ViewsForm.php b/core/modules/views/src/Form/ViewsForm.php
index 575b923..52d0fbe 100644
--- a/core/modules/views/src/Form/ViewsForm.php
+++ b/core/modules/views/src/Form/ViewsForm.php
@@ -166,7 +166,7 @@ public function buildForm(array $form, FormStateInterface $form_state, ViewExecu
     ];
 
     $form_object = $this->getFormObject($form_state);
-    $form += $form_object->buildForm($form, $form_state, $view, $output);
+    $form = array_merge($form, $form_object->buildForm($form, $form_state, $view, $output));
 
     return $form;
   }
