diff --git a/core/lib/Drupal/Core/Access/CsrfTokenGenerator.php b/core/lib/Drupal/Core/Access/CsrfTokenGenerator.php
index 3ba3d31..466078a 100644
--- a/core/lib/Drupal/Core/Access/CsrfTokenGenerator.php
+++ b/core/lib/Drupal/Core/Access/CsrfTokenGenerator.php
@@ -80,12 +80,9 @@ public function get($value = '') {
    *
    * @return bool
    *   TRUE for a valid token, FALSE for an invalid token.
-   *
-   * @deprecated in drupal:9.1.0 and is removed from drupal:10.0.0.
-   *   Not passing non-string $token variable.
    */
   public function validate($token, $value = '') {
-    if (!is_string($token)) {
+    if (empty($token) || !is_string($token)) {
       @trigger_error('Passing non-string $token to ' . __METHOD__ . '() is deprecated in drupal:9.1.0 and is removed
         from drupal:10.0.0. Refactor calling code. See https://www.drupal.org/node/3156880', E_USER_DEPRECATED);
       return FALSE;
@@ -94,8 +91,8 @@ public function validate($token, $value = '') {
     if (empty($seed)) {
       return FALSE;
     }
-
-    return hash_equals($this->computeToken($seed, $value), $token);
+    $value = $this->computeToken($seed, $value);
+    return hash_equals($value, $token);
   }
 
   /**
diff --git a/core/tests/Drupal/Tests/Core/Access/CsrfTokenGeneratorTest.php b/core/tests/Drupal/Tests/Core/Access/CsrfTokenGeneratorTest.php
index 1dbaadf..0041a52 100644
--- a/core/tests/Drupal/Tests/Core/Access/CsrfTokenGeneratorTest.php
+++ b/core/tests/Drupal/Tests/Core/Access/CsrfTokenGeneratorTest.php
@@ -6,6 +6,7 @@
 use Drupal\Tests\UnitTestCase;
 use Drupal\Core\Access\CsrfTokenGenerator;
 use Drupal\Component\Utility\Crypt;
+use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait;
 
 /**
  * Tests the CsrfTokenGenerator class.
@@ -15,6 +16,8 @@
  */
 class CsrfTokenGeneratorTest extends UnitTestCase {
 
+  use ExpectDeprecationTrait;
+
   /**
    * The CSRF token generator.
    *
@@ -174,14 +177,14 @@ public function providerTestValidateParameterTypes() {
    * @covers ::validate
    * @dataProvider providerTestInvalidParameterTypes
    *
-   * @expectedDeprecation Passing non-string $token is deprecated in drupal:9.1.0 and is removed
-   from drupal:10.0.0.
+   * @group legacy
    */
   public function testInvalidParameterTypes($token, $value = '') {
     $this->setupDefaultExpectations();
-
-    $this->expectException(\InvalidArgumentException::class);
-    $this->generator->validate($token, $value);
+    $this->throwException(new \InvalidArgumentException);
+    $this->expectDeprecation('Passing non-string $token to Drupal\Core\Access\CsrfTokenGenerator::validate() is deprecated
+      in drupal:9.1.0 and is removed from drupal:10.0.0. Refactor calling code. See https://www.drupal.org/node/3156880');
+    $this->assertFalse($this->generator->validate($token, $value));
   }
 
   /**
