diff --git a/core/lib/Drupal/Core/Access/CsrfTokenGenerator.php b/core/lib/Drupal/Core/Access/CsrfTokenGenerator.php
index 0614602..1d4812f 100644
--- a/core/lib/Drupal/Core/Access/CsrfTokenGenerator.php
+++ b/core/lib/Drupal/Core/Access/CsrfTokenGenerator.php
@@ -82,6 +82,11 @@ public function get($value = '') {
    *   TRUE for a valid token, FALSE for an invalid token.
    */
   public function validate($token, $value = '') {
+    if (!is_scalar($token)) {
+      @trigger_error('Passing non-string $token to ' . __METHOD__ . '() is deprecated in drupal:9.1.0 and is removed
+        from drupal:10.0.0. Refactor calling code. See https://www.drupal.org/node/3156880', E_USER_DEPRECATED);
+      return FALSE;
+    }
     $seed = $this->sessionMetadata->getCsrfTokenSeed();
     if (empty($seed)) {
       return FALSE;
