diff --git a/core/modules/comment/comment.module b/core/modules/comment/comment.module
index 23a81f1..c24b74d 100644
--- a/core/modules/comment/comment.module
+++ b/core/modules/comment/comment.module
@@ -668,18 +668,17 @@ function template_preprocess_comment(&$variables) {
  */
 function template_preprocess_field__comment(&$variables) {
   $element = $variables['element'];
-  if ($element['#field_type'] == 'comment') {
-    // Provide contextual information.
-    $variables['comment_display_mode'] = $element[0]['#comment_display_mode'];
-    $variables['comment_type'] = $element[0]['#comment_type'];
+  // Provide contextual information.
+  $variables['comment_display_mode'] = $element[0]['#comment_display_mode'];
+  $variables['comment_type'] = $element[0]['#comment_type'];
 
-    // Append additional attributes (eg. RDFa) from the first field item.
-    $variables['attributes'] += $variables['items'][0]['attributes']->storage();
+  // Append additional attributes (eg. RDFa) from the first field item.
+  $variables['attributes'] += $variables['items'][0]['attributes']->storage();
+
+  // Create separate variables for the comments and comment form.
+  $variables['comments'] = $element[0]['comments'];
+  $variables['comment_form'] = $element[0]['comment_form'];
 
-    // Create separate variables for the comments and comment form.
-    $variables['comments'] = $element[0]['comments'];
-    $variables['comment_form'] = $element[0]['comment_form'];
-  }
 }
 
 /**
