diff --git a/core/modules/user/tests/src/Functional/Views/UserFieldsAccessChangeTest.php b/core/modules/user/tests/src/Functional/Views/UserFieldsAccessChangeTest.php
index 10ac272..4d626f3 100644
--- a/core/modules/user/tests/src/Functional/Views/UserFieldsAccessChangeTest.php
+++ b/core/modules/user/tests/src/Functional/Views/UserFieldsAccessChangeTest.php
@@ -55,8 +55,8 @@ public function testUserFieldAccess() {
   }
 
   /**
-   * Tests the user name formatter shows a link to the user when there is
-   * access but not otherwise.
+   * Tests the user name formatter and shows a link to the user
+   when there is access not.
    */
   public function testUserNameLink() {
     $test_user = $this->drupalCreateUser();
@@ -66,7 +66,7 @@ public function testUserNameLink() {
       'title' => 'View user profile.',
       'class' => 'username',
     ];
-    $link = $test_user->toLink(NULL, 'canonical', ['attributes' => $attributes])->toString();
+    $test_user->toLink(NULL, 'canonical', ['attributes' => $attributes])->toString();
 
     // No access, so no link.
     $this->drupalGet('test_user_fields_access');
